unidist/cli/backends/dask/runner.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (C) 2021 Modin authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""``DaskRunner`` class functionality."""
+
+import os
+import warnings
+
+from unidist.cli.backends.base.runner import BackendRunner
+from unidist.cli.backends.utils import Defaults, BackendName, validate_num_cpus
+
+
+class DaskRunner(BackendRunner):
+    """
+    An implementation of unidist ``BackendRunner`` for Dask backend.
+
+    Parameters
+    ----------
+    **cli_kwargs : dict
+        Keyword arguments supported by unidist CLI.
+    """
+
+    def __init__(self, **cli_kwargs):
+        self.backend = BackendName.DASK
+        super().__init__(**cli_kwargs)
+
+    def check_kwargs_support(self, **kwargs):
+        """Check the support of `kwargs` combination for Dask backend."""
+        hosts = kwargs.get("hosts", Defaults.HOSTS)
+        num_cpus = kwargs.get("num_cpus", Defaults.NUM_CPUS)
+        if hosts == Defaults.HOSTS:
+            self.hosts = None
+            if num_cpus == Defaults.NUM_CPUS:
+                self.num_cpus = validate_num_cpus([num_cpus])[0]
+            elif isinstance(num_cpus, list) and len(num_cpus) == 1:
```

Comment:
If we pass a single value to a command line parameter, isn't that a string?

Reply:
`hosts` and `num_cpus` are accumulating arguments. In case value (one or more) is provided, it will be a list from (one or more) values

Reply:
Ok, I see